docs/data/material/components/chips/ChipMaterialYouPlayground.js

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Chip from '@mui/material-next/Chip';
+import MaterialYouUsageDemo from 'docs/src/modules/components/MaterialYouUsageDemo';
+
+export default function ChipMaterialYouPlayground() {
+  return (
+    <MaterialYouUsageDemo
+      componentName="Chip"
+      data={[
+        {
+          propName: 'variant',
+          knob: 'select',
+          defaultValue: 'filled',
+          options: ['filled', 'outlined', 'elevated'],
+        },
+        {
+          propName: 'color',
+          knob: 'select',
```

Comment:
```suggestion
          knob: 'select',
          defaultValue: 'primary',
```

So that on the initial render the select is not empty.

Reply:
When no color prop is provided, the specs default is used:

- secondary for the filled chip
- outline for the outlined chip
- surfaceContainerLow for the elevated chip

An empty selection is valid.

From MY specs, different variants support different colors (filled supports accent colors, outlined only outline, elevated supports accent and surface colors) this becomes a harder issue than it looks like. This calls for a separate issue to figure out the best API. I will open these follow-up issues after this PR is merged.